pkg/kubectl/cmd/wait/wait.go

```diff
@@ -300,7 +305,12 @@ func IsDeleted(info *resource.Info, o *WaitOptions) (runtime.Object, bool, error
 }
 
 func isDeleted(event watch.Event) (bool, error) {
-	return event.Type == watch.Deleted, nil
+	switch event.Type {
```

Comment:
Can you rename the function to `isDeletedOrError` currently the name and the code don't express the same thing.

Reply:
This only returns true on deleted.